addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -35,9 +35,9 @@
                 </div>
             </div>
             <div class="dropdown d-none d-lg-block">
-                <a role="button" href="#" data-bs-toggle="dropdown" t-attf-class="dropdown-toggle btn btn-light #{'d-none' if not partners else ''}" aria-expanded="true" aria-label="Open countries dropdown">
+                <a role="button" href="#" data-bs-toggle="dropdown" class="dropdown-toggle btn btn-light" aria-expanded="true" aria-label="Open countries dropdown">
                     <t t-foreach="countries" t-as="country">
-                        <span t-if="partners and country['active']" t-out="country['country_id'][1]"/>
```

Comment:
Same. Note that another idea to fix this without changing the template is to *add* nodes which set `partners` to `True` before those nodes. Probably the best actually.